tools-local/tasks/core-setup.tasks.csproj

```diff
@@ -14,9 +14,7 @@
     <PackageReference Include="NuGet.ProjectModel">
       <Version>$(NugetProjectModelPackageVersion)</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.DependencyModel">
-      <Version>1.1.1</Version>
-    </PackageReference>
+    <ProjectReference Include="..\..\src\managed\Microsoft.Extensions.DependencyModel\Microsoft.Extensions.DependencyModel.csproj"/>
```

Comment:
You cannot take a project reference here, because it will cause a cycle (i.e. we cannot build a project in core-setup until these build tasks are built). 

Instead you need to update to a later package version.